features/telemetry/listeners/flow/src/main/java/org/opennms/netmgt/telemetry/listeners/flow/ipfix/TcpListener.java

```diff
@@ -85,6 +85,8 @@ protected void initChannel(final SocketChannel ch) {
                                 .addLast(new ChannelInboundHandlerAdapter() {
                                     @Override
                                     public void exceptionCaught(final ChannelHandlerContext ctx, final Throwable cause) throws Exception {
+                                        cause.printStackTrace();
```

Comment:
Oops.

Reply:
Oh, this wasn't in my original branch. This was added during the rebase. I'm checking this...